internal/datastore/crdb/pool/balancer.go

```diff
@@ -228,6 +227,10 @@ func (p *nodeConnectionBalancer[P, C]) pruneConnections(ctx context.Context) {
 			continue
 		}
 
+		// var limitedPruneCount uint32
+		// for limitedPruneCount = 0; p.limiter.Allow() && limitedPruneCount < numToPrune; limitedPruneCount++ {
+		// }
+
```

Comment:
remove leftover?

Reply:
yep, thought I did.

I originally played with limiting the pruner but found that limiting on connection eclipsed it.